install/helm/agones/templates/service/allocation.yaml

```diff
@@ -47,6 +47,9 @@ spec:
     - port: {{ .Values.agones.allocator.service.grpc.port }}
       name: {{ .Values.agones.allocator.service.grpc.portName }}
       targetPort: {{ .Values.agones.allocator.service.grpc.targetPort }}
+      {{- if .Values.agones.allocator.service.grpc.appProtocol }}
+      appProtocol: {{.Values.agones.allocator.service.grpc.appProtocol}}
```

Comment:
This is the correct approach - but we'll also need a `.Values.agones.allocator.service.http.appProtocol` in the port section above (line 37 onwards).